### PR TITLE
Adding minutes

### PR DESCRIPTION
**Pull request by teresanunez**

### CODE REVIEW

File: 20200326/MINUTES.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        - [ ] 2nd Ctrl+C abort the macro: status update
+	     No update
+	- [ ] Zibi will review the last debug information from Thorsten
+	     Zibi analysed the last logs, PR for dumping status information of the counter/timers is done.
+	     See discussion in PR#1308
+	- [ ] Juliano will share the code of the proxy/mirror controller that they use
+	     Aureo will provide the code after the meeting.
+    * How Tango issues affects Sardana?
+        * https://github.com/tango-controls/pytango/issues/292 and https://github.com/tango-controls/cppTango/issues/686
+	 Zibi advanced in the workarounds in tango for that. The bug affects the 4th beamline migrating
```
Comment (reszelaz) [inline 20200326/MINUTES.md]: Workarounds in Taurus. These were propsed as WIP PR to Taurus.

Comment (reszelaz) [reply 20200326/MINUTES.md]: You alread say it is to Taurus in the next sentences.

File: 20200326/MINUTES.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	 to python3. It would be imposible to migrate it to python3 if this is not solved.
+	 Three workarounds are evaluated (PRs to Taurus). One seems to solve the problem.
+	 cpptango has a PR to solve the issue and will be evaluated.
+	 Could be that the PRs for Taurus are not needed any more.
+        * https://github.com/tango-controls/pytango/issues/307 and https://github.com/sardana-org/sardana/pull/1298
+	 Jairo is working on this, it is not easy to solve in pytango, probably it has to be done in cpptango and
+	 see how logs are constructed.
```
Comment (reszelaz) [inline 20200326/MINUTES.md]: Typo: locks instead of logs.

File: 20200326/MINUTES.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    * Several PMTV and macroexecutor/sequencer improvement PRs
+        No update
+    * showscan online improvements
+        There are two PRs related. One changing the way the trends are updated (it was done periodically and now is based
```
Comment (reszelaz) [inline 20200326/MINUTES.md]: The other way around: before - on events - now periodically.

File: 20200326/MINUTES.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+	on events, this PR is integrated. In other one Tiago works in data inspection for showscan online (work in progress).
+    * Jan20 release:
+	Some PRs have been integrated but there are still 8 PRs to be reviewed and merged. They have been assigned.
+    * sardana-org/sardana#1315
+         A decision is postponed and will be done in the PR
```
Comment (reszelaz) [inline 20200326/MINUTES.md]: Here we said that we will all think about it and reply in the issue. PR will be created afterwards when we agree on what to remove and what will stay.

**Review by reszelaz — APPROVED** [review]
Great! Very complete as always! I just made some tiny comments.

**Comment by teresanunez** [comment]
Thanks, ZIbi, I have added your comments.

**Review by aureocarneiro — APPROVED** [review]

**Pull request merged**